pallet-logion-loc/src/lib.rs

```diff
@@ -183,6 +234,21 @@ pub mod pallet {
 
         /// Query for checking that a signer is a legal officer
         type IsLegalOfficer: IsLegalOfficer<Self::AccountId, Self::RuntimeOrigin>;
+
+        /// Token Record identifier
+        type TokensRecordId: Member + Parameter + Default + Copy;
```

Comment:
Do we really need a parametrizable type `TokensRecordId` ?
I feel like simple declaration is sufficient `pub type TokensRecordId = H256;`

Reply:
I just followed the same principle as for `LocId` and `CollectionItemId` which are both parameters. In general, when writing pallets, it is considered a good practice to do so so and make choices on runtime side.